cite/index.rst

```diff
@@ -19,6 +19,11 @@ The articles on GMT are:
   The Generic Mapping Tools and Animations for the Masses.
   *Geochemistry, Geophysics, Geosystems*, 25(6).
   https://doi.org/10.1029/2024GC011545
+* **Origin of GMT**:
+  Wessel, P. (2024). 
+  The origins of the generic mapping tools: From table tennis to geoscience. 
+  *Perspectives of Earth and Space Scientists*, 5, e2023CN000231. 
+  https://doi.org/10.1029/2023CN000231
```

Comment:
Maybe add the issue even it is not included in the recommended citation (see "How to cite" at https://agupubs.onlinelibrary.wiley.com/action/showCitFormats?doi=10.1029%2F2023CN000231), but for completeness and consistency.

```suggestion
* **Origin of GMT**:
  Wessel, P. (2024). 
  The origins of the generic mapping tools: From table tennis to geoscience. 
  *Perspectives of Earth and Space Scientists*, 5(1), e2023CN000231. 
  https://doi.org/10.1029/2023CN000231
```